Doxygen/doxygen_footer.html

```diff
@@ -4,7 +4,7 @@
   </div>
   <div id="copyright">
    <p>
-     &copy; 2014 Google
+     &copy; 2015 Google
```

Comment:
We only want to change the year for files that have been changed, which would be tricky to achieve automatically.